### PR TITLE
change log

### PR DESCRIPTION
**Pull request by dbenchi**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [master]
 ### Added
 
+## [0.0.8] - 2016-01-18
+### Added
+- Support setMailboxes #20
```
Comment (ddolcimascolo) [inline CHANGELOG.md:12]: Please be more precise:
- Added Client.createMailbox
- Added Client.destroyMailbox
  etc..

Comment (dbenchi) [inline CHANGELOG.md:12]: Done

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,19 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
+- Add CreateMailboxAck class
+- Add MailboxesSet class
+- Add Client.setMailboxes
+- Add Client.createMailbox
+- Add Client.updateMailbox
+- Add Client.destroyMailbox
+- Add Client._handleMailboxesSetResponse
```
Comment (ddolcimascolo) [inline CHANGELOG.md]: Remove this, it's internal :)

Comment (dbenchi) [inline CHANGELOG.md]: Done

**Comment by ddolcimascolo** [comment]
:+1:

**Pull request merged**